src/assets/i18n/en.json

```diff
@@ -1,8 +1,8 @@
-    "ABOUT_ZONEMASTER_TEXT": "Give your domain a complete physical! Zonemaster helps you to control how your website is doing. As a bonus, you also get a better understanding of how the domain name system, DNS, works. When a domain (also called zone) is sent to Zonemaster, the program investigates the state of the domain from beginning to end. This is done through Zonemaster examining DNS from the root (.) to the TLD (top-level domain, for example, .se), and then finally through the DNS servers that contain information about the specified domain (for example, zonemaster.se). Zonemaster is also able to perform many other tests, for example, checking the DNSSEC signatures, that different hosts can be accessed and that the IP addresses are valid. This is all to make sure that your domain runs as good as possible",
+    "ABOUT_DNS_TEXT": "Most people who use the Internet have never reflected on what actually happens when one types an address in their web browser and then clicks the 'Enter' button. In short, the DNS (Domain Name System) is a database that associates domain names to IP (Internet Protocol) addresses, the same way a phone book associates names to phone numbers. An IP address is a unique series of numbers that identifies every computer that is connected to the Internet. It’s similar to the way every telephone has its own number in the telephone network. Such long series of numbers work fine for computers, but for people it is way more difficult to memorize. Thus, the DNS has been developed as an extra layer to map between names that we can remember and IP addresses used by computers whenever we want to reach a web server on a specific site or use other services on the Internet (e.g., mail).",
+    "ABOUT_ZONEMASTER_TEXT": "Give your domain a complete checkup! Zonemaster helps you assess how your domain is doing. Besides, you also get a better understanding of how the DNS (Domain Name System) works. When a domain name is sent to Zonemaster for testing, the program investigates the state of the domain (sometimes called 'zone') from top to bottom of the DNS tree. Specifically, this is usually done by examining DNS data from the root ('.') to the TLD (Top-Level Domain; e.g. '.net') for that domain, and then finally through the DNS servers that contain authoritative information about the specified domain (e.g., 'zonemaster.net'). Zonemaster is also able to perform many other tests, such as checking DNSSEC signatures, or that different hosts can be accessed and that IP addresses are valid. This is all to make sure that your domain runs as smoothly as possible.",
     "About Zonemaster": "About Zonemaster",
-    "About domain name system, DNS": "About domain name system, DNS",
+    "About domain name system, DNS": "About DNS",
```

Comment:
I cannot see why key and value are differenth here. I suggest:
```
"About DNS": "About DNS",
```

Reply:
Changing the key has other implications elsewhere in the code. Also that behavior will change when #349 is merged, so will not update for now.